gcalendarsync.js

```diff
@@ -300,13 +337,9 @@ function syncToCalendar() {
       if (eventIdx >= 0) {
         calEventIds[eventIdx] = null;  // Prevents removing event below
         var calEvent = calEvents[eventIdx];
-        if (eventMatches(calEvent, sheetEvent)) {
-          addEvent = false;
-        } else {
-          // Delete and re-create event. It's easier than updating in place.
-          calEvent.deleteEvent();
-          numUpdated++;
-        }
+        if (!eventMatches(calEvent, sheetEvent)) {
+          //update the event
+          updateEvent(calEvent, sheetEvent);
```

Comment:
Hi dave
how can i get the full working code ?
i m new to github,,,,
i see you made some changes,,,

thank you!!!